serveradmin/servershell/views.py

```diff
@@ -162,6 +162,9 @@ def get_results(request):
             'message': str(error)
         }))
 
+    # Query successful term must be valid we can save it.
```

Comment:
This sentence is not completely understandable for me. I guess you meant it like that:
```suggestion
    # Query successful term must be valid here, so we can save it safely now.
```